.github/workflows/nix.yml

```
@@ -0,0 +1,23 @@
+---
+name: "nix-build"
+on:
+  pull_request:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.4.0
+      - uses: cachix/install-nix-action@v16
+        with:
+          nix_path: nixpkgs=channel:nixos-unstable
+          extra_nix_config: |
+            trusted-public-keys = hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ= cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
+            substituters = https://hydra.iohk.io https://cache.nixos.org/
+      - uses: cachix/cachix-action@v10
+        with:
+          name: tweag
+          authToken: "${{ secrets.TWEAG_CACHIX_AUTH_TOKEN }}"
```

@purefn Try the following (disclaimer: I don't know what I'm doing):
```suggestion
          authToken: "${{ secrets.TWEAG_CACHIX_AUTH_TOKEN }}"
          signingKey: '${{ secrets.CACHIX_SIGNING_KEY }}'
```

---

> @purefn Try the following (disclaimer: I don't know what I'm doing):

Giving it a whirl. I just wasn't sure what the name of variable might be. I can try prefixing it with `TWEAG_` if this doesn't work.